src/main/java/edu/cornell/kfs/paymentworks/util/PaymentWorksUtil.java

```diff
@@ -40,8 +41,10 @@
 import edu.cornell.kfs.paymentworks.xmlObjects.PaymentWorksFieldChangesDTO;
 
 public class PaymentWorksUtil {
```

Comment:
Given that this is making use of other services, but can't use dependency injection, I would consider converting it to a service as well.